ui/src/setupTests.ts

```diff
@@ -1,9 +1,13 @@
+import { configure } from "@testing-library/react";
+import "@testing-library/jest-dom";
 import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
 import Enzyme from "enzyme";
 import { enableFetchMocks } from "jest-fetch-mock";
 
 Enzyme.configure({ adapter: new Adapter() });
 
+configure({ testIdAttribute: "data-test" });
```

Comment:
Do you know if this also maintains functionality for `data-testid` ?

Reply:
No. This will need to be removed once https://github.com/canonical-web-and-design/maas-ui/pull/3295 is merged to use the default test id ('data-testid').